src/common/components/facilitator/ride.tsx

```diff
@@ -358,8 +409,11 @@ class Ride extends Component<Props, State> {
     const disabled =
       this.state.originalRideState &&
       this.state.originalRideState.status !== 'OPEN';
+    const confirmstatus= !dateInFuture && this.state.originalRideState?.status === 'CONFIRMED';
```

Comment:
You could probably just join this logic onto `cannotReopen` rather than having its own variable.

Reply:
thanks!